.webpack/webpack.dev.mjs

```diff
@@ -39,7 +39,7 @@ export default merge(common, {
         return shouldWrite;
       }
     },
-    watchFiles: ['**/*.css'],
+    watchFiles: ['src/**/*.css', 'example/**/*.css'],
```

Comment:
Webpack was rebuilding when playwright test reports were being generated and updated so let's be explicit